src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Smart Bulletin No. 036"
+intro: "Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies” Act or “CHARGE” Act)."
```

Comment:
Use of GSA SmartPay® Charge Cards to Pay for Electric Vehicle Charging Pursuant to the “Charging Helps Agencies Realize General Efficiencies Act” or “CHARGE Act”).